src/components/buttons/secondary-icon-button/secondary-icon-button.js

```diff
@@ -59,24 +18,30 @@ export const SecondaryIconButton = props => {
       label={props.label}
       onClick={props.onClick}
       isDisabled={props.isDisabled}
+      css={theme => getBaseStyles(theme, props)}
     >
-      <IconContainer isDisabled={props.isDisabled}>
-        {React.cloneElement(props.icon)}
-      </IconContainer>
+      {props.icon &&
+        React.cloneElement(props.icon, {
+          size: props.size,
```

Comment:
I don't think this is right. Before you could just use the size prop from the Icon itself, like in CalendarHeader. We should do `props.icon.size || props.size`

Reply:
I decided to remove the size prop altogether since after all we could already change the size on the `Icon` element that is passed as a prop.